py3status/modules/online_status.py

```diff
@@ -21,13 +20,7 @@
 @author obb
 """
 
-import os
-import subprocess
-try:
-    # python3
-    from urllib.request import urlopen
-except:
-    from urllib2 import urlopen
+from urllib.parse import urlsplit
```

Comment:
```
lasers~/src/py3status/py3status/modules git:(fix-dpms✗) py3status-python2 -l                                              
usage: py3status-python2 [-h] [-b] [-c I3STATUS_CONF] [-d] [-i INCLUDE_PATHS]
                         [-l LOG_FILE] [-n INTERVAL] [-s] [-t CACHE_TIMEOUT]
                         [-v]
py3status-python2: error: argument -l/--log-file: expected one argument
```
```
lasers~/src/py3status/py3status/modules git:(fix-dpms✗) py3status-python2 -c ~/.config/i3/i3status-play.conf 
{"click_events": true, "stop_signal": 20, "version": 1}
[[]
,[{"color": "#FF0000", "instance": "", "name": "dpms", "full_text": "DPMS"},{"color": "#FF0000", "instance": "", "name": "online_status", "full_text": "Ge \u25a0"},{"color": "#ffffff", "instance": "", "name": "static_string", "full_text": "APPLE"}]
^C%
```
Okay. That works. Just no test modules.

> I'm sure there is always more, anyhow we should try to finish you outstanding PRs

Agreed. I wanted them all out. This one should finished, I think.
Few issues will need to be closed too (ie rainbow).
Some PRs may be finished. Pending on strings or missing tags (transformed).